OJP/OJP_Trips.xsd

```diff
@@ -108,7 +108,12 @@
 			</xs:element>
 			<xs:element name="OptimisationMethod" type="OptimisationMethodEnumeration" minOccurs="0">
 				<xs:annotation>
-					<xs:documentation>the types of algorithm that can be used for planning a journey (fastest, least walking, etc)</xs:documentation>
+					<xs:documentation>The types of algorithm that can be used for planning a journey (fastest, least walking, etc)</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="AccessibilityRoutingPreference" type="AccessibilityRoutingPreferenceEnumeration" minOccurs="0" maxOccurs="unbounded">
+				<xs:annotation>
+					<xs:documentation>If bestAccessibility is chosen as OptimisationMethod, the algorithm tries to minimise obstacles and sections without accessibility aids. The user may specify the criteria to be considered. The order of priority is given by the sequence in which they are listed (starting with the highest priority). Higher priority means, in terms of a cost function, that a change at a higher priority can never be compensated by changes at the lower priorities.</xs:documentation>
```

Comment:
This is a very interesting statement. I would like to discuss it.

Reply:
Same for me.